packages/ui/src/Select/CheckableSelectField.module.scss

```diff
@@ -0,0 +1,36 @@
+@use './styles/constants' as c;
```

Comment:
Seems like the checkbox is higher than the text
![image](https://user-images.githubusercontent.com/13898415/127179222-a3cec08a-a16b-4c59-853c-933d9476d098.png)